wdfn-server/waterdata/templates/partials/monitoring_location_header.html

```diff
@@ -0,0 +1,93 @@
+{% extends 'partials/header.html' %}
```

Comment:
A somewhat subtle thing with Jinja templating is that if you "include" a template you can not then override a block in the template where you included the partial. So in order to add page_specific_menus, I had to extend (inherit) the partials/header.html and override page_specific_menus in that. 

This means some repetition of template code in monitoring_location.html but it is minimal and consists mostly of including other partials. There was already a lot of overriding in that template.